### ISSUE TITLE
Timeline error: fetch error issue. Does anyone else have it too?

### ISSUE THREAD

**Issue by rarelygoeshere** [furyutei/twMediaDownloader#130]
I just encountered this error today while using Twitter Media Downloader. I was wondering what could have caused this and though I wonder if this project is ever getting new updates, I'll post the issue to bring it to awareness and ask for help anyway.
![screenshot_1692759505](https://github.com/furyutei/twMediaDownloader/assets/125143953/2b02fd40-b552-4476-8bda-84de94d17ee5)

**Comment by mysteriousbenefactor** [comment]
i'm having the same issue, not sure how to fix it.

**Comment by ajayraho** [comment]
yep, same here

**Comment by ANaPhOrAX** [comment]
me as well

**Comment by 06000208** [comment]
getting this as well, twitter likely changed something again

in my case, though, it does successfully download some amount of images before occurring, and if i try to start the download at the timestamp where it stops, it will get the error right away

**Comment by Zkuader** [comment]
I have the same error, it is impossible to download more, unfortunately it has a limit,

(*) Timeline error: fetch error
--------------------------------------------------------------------------------
[Complete(timeline-error)]  -   ( Tweet: 0 / Media: 0 )

**Comment by 06000208** [comment]
For me, it looks like it's downloading all available images, before encountering the error. The number of posts line up with the number in the top left of the media tab. As far as I've tested, anyway.

**Comment by 06000208** [comment]
ah, no, found an account where it got the error after 800ish posts, making this issue highly resemble my experience with #115

**Comment by Nikolai521** [comment]
Same here, It only happens when you have the rtwit option enabled
sometimes it gets complicated for those people who dont use the "media" tab and just retwitts his works

**Comment by 06000208** [comment]
I've been getting it without "include rts" enabled

**Comment by CmdrShepardsPie** [comment]
> I've been getting it without "include rts" enabled

Yes, this stopped working for me in the last few days or weeks, and disabling "include rts" seems to have "fixed" it for me, though of course now I don't get RTs. Thanks for the tip!

**Comment by rarelygoeshere** [comment]
Man with all of the recent changes and this project not updating since last year we might need someone who's willing to fork it and make a new Twitter Media Downloader for us users.

**Comment by 06000208** [comment]
> > I've been getting it without "include rts" enabled
> 
> Yes, this stopped working for me in the last few days or weeks, and disabling "include rts" seems to have "fixed" it for me, though of course now I don't get RTs. Thanks for the tip!

No, I mean, I have rts disabled and I'm still getting the error sometimes.

**Comment by Nikolai521** [comment]
> > > I've been getting it without "include rts" enabled
> > 
> > 
> > Yes, this stopped working for me in the last few days or weeks, and disabling "include rts" seems to have "fixed" it for me, though of course now I don't get RTs. Thanks for the tip!
> 
> No, I mean, I have rts disabled and I'm still getting the error sometimes. Glad it seemed to work for you though.

Same, now im having the error betwheen 30-40 posts or 2 months 
![image](https://github.com/furyutei/twMediaDownloader/assets/25958695/b7d92b0a-0a23-41f5-be08-3af1f71b615c)

**Comment by ruben1PvP** [comment]
I'm not updated in Twitter API changes, but this seems that, as InMirrors said, they change it so you can't interact with tweets older than a certain point in time. Change a setting or not won't change a thing. If that's the case, or we use another API or we have to wait until they lift this restriction or someone manages to bypass it. Knowing this software is discontinued, we can only hope the former happens.

**Comment by ruben1PvP** [comment]
> I'm not updated in Twitter API changes, but this seems that, as InMirrors said, they change it so you can't interact with tweets older than a certain point in time. Change a setting or not won't change a thing. If that's the case, or we use another API or we have to wait until they lift this restriction or someone manages to bypass it. Knowing this software is discontinued, we can only hope the former happens.

Update: They seem to have killed all old versions of the APIs, so this extension officially died. No tweets can be downloaded whatsoever.

**Comment by rarelygoeshere** [comment]
> > I'm not updated in Twitter API changes, but this seems that, as InMirrors said, they change it so you can't interact with tweets older than a certain point in time. Change a setting or not won't change a thing. If that's the case, or we use another API or we have to wait until they lift this restriction or someone manages to bypass it. Knowing this software is discontinued, we can only hope the former happens.
> 
> Update: They seem to have killed all old versions of the APIs, so this extension officially died. No tweets can be downloaded whatsoever.

Dammit! No wonder it can't download anything.

I honestly don't know how to code, but I went ahead and [forked the repo](https://github.com/rarelygoeshere/twMediaDownloader) anyway. Just to see if anyone's willing to instruct me to allow for it to become its own project and keep it alive for me.

I just hope when the repo goes private it wont affect the fork. If so I would look into downloading it locally.
